### PR TITLE
Missing things resilience

### LINKED ISSUE #2: Error when attempting to run script
```
> call scripts\garbage-collector\garbo.js

Using adventure value 5000.
Collecting garbage!

Autoattack is disabled.
Preference currentMood changed from default to apathetic
Preference customCombatScript changed from WHAM to garbo
CCS set to garbo
JavaScript exception: No stashClan set. (file:/D:/KoLmafia1/scripts/garbage-collector/garbo.js#5153)
at file:/D:/KoLmafia1/scripts/garbage-collector/garbo.js:5153 (StashManager)
at file:/D:/KoLmafia1/scripts/garbage-collector/garbo.js:5122 (withStash)
at file:/D:/KoLmafia1/scripts/garbage-collector/garbo.js:4016 (main)
```

### LINKED ISSUE #5: Fails trying to acquire a defective Game Grid token
```
Visiting the Boxing Daycare

[7771] Boxing Daycare
Preference     lastEncounter changed from The Mad Tea Party to Visit the Boxing Day Spa
Encounter:     Visit the Boxing Day Spa
Preference _daycareSpa changed from false to     true
You acquire an effect: Uncucumbered (100)
Searching for     "defective Game Grid token"...
Search complete.
Desired     purchase quantity not reached (wanted 1, got 0)
Using cached search     results for defective Game Grid token...
Desired purchase quantity not     reached (wanted 1, got 0)
You need 1 more defective     Game Grid token to continue.
JavaScript     exception: Script interrupted.     (file:/D:/KoLmafia1/scripts/garbage-collector/garbo.js#44602)
at     file:/D:/KoLmafia1/scripts/garbage-collector/garbo.js:44602 (f)
at     file:/D:/KoLmafia1/scripts/garbage-collector/garbo.js:45003 (freeFights)
at     file:/D:/KoLmafia1/scripts/garbage-collector/garbo.js:45328
at     file:/D:/KoLmafia1/scripts/garbage-collector/garbo.js:45723 (withStash)
at     file:/D:/KoLmafia1/scripts/garbage-collector/garbo.js:45317 (main)
```

running d7acae8

### PR DESCRIPTION
**Pull request by Phillammon**
- Do not attempt to fax Embezzler if we lack a VIP key to access a fax machine. WARNING: THIS TURNS OFF BASICALLY ALL DAILY FIGHTS, WILL FIND A PATCH SOON I PROMISE
- Use pokegarden if available for actual fights
- Don't abort if our clan is missing zarara or a pool table,
- Don't abort if we don't own BB, an asdon, a defective GG token, a Robortender, a KramCo or a Witchess board. Resolves #5.
- Don't abort if we don't have Bind Lasgmbie, Ancestral Recall, Snokebomb, Feel Hatred, or Sweet Synthesis.
- Don't always abort if we don't have a stashclan set (Userconfirm allows it to be automatically set to the user's current clan, or abort to set manually). Resolves #2.
- Equip retrocape and an ice nine if we aren't a seal clubber and don't have a haiku katana
- Accept a broader array of possible ways of breathing underwater
- Don't get beaten up by blood bond.
- Use glark cables for free kills

### CODE REVIEW

File: src/fights.ts [hunk 1]
```diff
@@ -63,95 +64,99 @@ import { withStash } from "./stash";
 
 export function dailyFights() {
   meatMood(true).execute(myAdventures() * 1.04 + 50);
+  if (have($item`clan VIP key`)) {
```
Comment (phulin) [inline src/fights.ts:67]:
```suggestion
  if (have($item`Clan VIP Lounge key`)) {
```

File: src/fights.ts [hunk 2]
```diff
@@ -63,95 +64,99 @@ import { withStash } from "./stash";
...
-  if (
-    (!have($item`photocopied monster`) || get("photocopyMonster") !== embezzler) &&
-    !get("_photocopyUsed")
-  ) {
-    faxbot(embezzler, "CheeseFax");
-  }
-
-  while (get("_poolGames") < 3) cliExecute("pool aggressive");
+    if (getClanLounge()["Clan pool table"] !== undefined)
+      while (get("_poolGames") < 3) cliExecute("pool aggressive");
```
Comment (phulin) [inline src/fights.ts:77]:
```suggestion
    if (getClanLounge()["Clan pool table"] !== undefined) {
      while (get("_poolGames") < 3) cliExecute("pool aggressive");
    }
```

File: src/fights.ts [hunk 3]
```diff
@@ -63,95 +64,99 @@ import { withStash } from "./stash";
...
-        puttyCount++;
-      }
-      set("spookyPuttyCopiesMade", 5);
-    });
+        set("spookyPuttyCopiesMade", 5);
+      });
+    }
   }
+  if (have($item`Packet of tall grass seeds`)) use($item`Packet of tall grass seeds`);
+  if (have($item`unwrapped retro superhero cape`)) cliExecute("retrocape robot kill");
```
Comment (phulin) [inline src/fights.ts:159]: these checks should both be in the main adventure loop, not here

File: src/fights.ts [hunk 4]
```diff
@@ -367,6 +372,7 @@ const freeFightSources = [
         SourceTerminal.educate([$skill`Extract`, $skill`Portscan`]);
       }
       adventureMacro($location`Your Mushroom Garden`, Macro.trySkill("Portscan").meatKill());
+      if (have($item`Packet of tall grass seeds`)) use($item`Packet of tall grass seeds`);
```
Comment (phulin) [inline src/fights.ts:375]:
```suggestion
      if (have($item`packet of tall grass seeds`)) use($item`packet of tall grass seeds`);
```
nitpick sorry

File: src/index.ts [hunk 5]
```diff
@@ -142,11 +146,22 @@ function dailySetup() {
...
+    myThrall() !== $thrall`Lasagmbie` &&
+    haveSkill($skill`Bind Lasagmbie`)
+  ) {
     useSkill($skill`Bind Lasagmbie`);
   }
 
-  if (!get("_clanFortuneBuffUsed")) cliExecute("fortune buff meat");
+  if (
+    !get("_clanFortuneBuffUsed") &&
+    have($item`clan VIP key`) &&
```
Comment (phulin) [inline src/index.ts:159]:
```suggestion
    have($item`Clan VIP lounge key`) &&
```

**Review by phulin — DISMISSED** [review]

File: src/index.ts (hunk 5 above)
Comment (Phillammon) [reply src/index.ts]: Resolved as of b9f33ec

File: src/fights.ts (hunk 4 above)
Comment (Phillammon) [reply src/fights.ts:373]: Resolved as of b9f33ec

File: src/fights.ts (hunk 3 above)
Comment (Phillammon) [reply src/fights.ts]: Resolved as of b9f33ec

File: src/fights.ts (hunk 2 above)
Comment (Phillammon) [reply src/fights.ts]: Resolved as of b9f33ec

File: src/fights.ts (hunk 1 above)
Comment (Phillammon) [reply src/fights.ts]: Resolved as of b9f33ec

**Review by phulin — APPROVED** [review]

**Pull request merged**